README.md

```diff
@@ -4,9 +4,14 @@
 
 > A simple App that allows a user to make a to-do-list.
 
+![screenshot](./screenshot.JPG)
 ## Built With
 
 - Javascript, HTML and CSS
+
+## Live Demo
+
+[Live Demo Link](https://medaminedev66.github.io/to-do-list/)
 ## Getting Started
 
 - In terminal or Command line. Navigate to a directory in your local storage where you'd like to store the project and paste ```https://github.com/medaminedev66/to-do-list.git```
```

Comment:
- [ Optional] Kindly try to update the `Live link` and correct the information for getting started in your README file. Like `npm install`, `npm run build` and `npm start` are some of the information missing to start a project locally.